### ISSUE TITLE
Problem in reload packages & chats

### ISSUE THREAD

**Issue by BolaNasr** [threefoldtech/js-sdk#888]
### Search first
![image](https://user-images.githubusercontent.com/10259267/91973606-ce0c4800-ed1c-11ea-99a0-b0d62a30e1e4.png)

### Description
When try to reload package , it's doesn't reload because it's have a check if a package already exist get it from system not from path with new change .

### Version information
* OS: e.g. Ubuntu 18.04

### Steps to reproduce
- start Threebot
- get gedis client 
- call any function in actor package
- change in actor package 
- call add_pacakges in admin_package to reload
- call the same function in actor package should changed

**Comment by abom** [comment]
Current @BolaNasr PR would work only with chats (if we reloaded a package, then restarted the chat it would work).

**Comment by abom** [comment]
Currently, we need to update the loading of gedis actors to work, will move this to alpha4 for now.

**Comment by OldhamMade** [comment]
Added an upvote for this: I need it for reloading Actors to avoid having to restart the Threebot often.

Preferred workflow would be to update an actor file, then inform the gedis server (either via a `j.clients.gedis` or `j.servers.gedis`) that the module needs to be reloaded.

I tested a quick custom implementation similar to `j.tools.codeloader.load_python_module` _without_ the two lines pictured above, but the module doesn't change.
